_RFCs/0007-deprecating-packages.md

```diff
@@ -0,0 +1,62 @@
+- Change Name: deprecate_unsupported_software_plans
+- Start Date: 2018-05-07
+- RFC PR: (leave this empty)
+- Core-Plans Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+When a piece of software that is packaged with a Habitat plan is no longer supported by the manufacturer of that software, we need a process to deprecate that plan.
+
+Deprecating a plan does NOT involve removing it from builder. Rather, we must indicate to users that the plan will no longer be updated by the Habitat team except in cases of security vulnerabilities.
```

Comment:
Wouldn't by definition the plan no longer be updated? If the manufacturer has ceased support, then how can the Habitat team update the package? Does this refer to updating transitive dependencies? To what extent will support be offered?

I think either the language around updating packages should be removed or clarification about the scope of updates should be enumerated.

Stronger language version:

> Deprecating a plan does will not remove it from Builder. Instead, we will indicate to users that the plan will no longer be updated.

Reply:
In some cases we have attempted to patch older software that is no longer maintained by the manufacturer (something I would prefer we cease to do except in extenuating circumstances). Additionally, even when a manufacturer no longer officially supports old software, they do sometimes update it in the case of extreme security vulnerabilities.